Assets/Samples/2 - Microphone/MicrophoneRecord.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace Whisper.Samples
```

Comment:
Move it to Package/Whisper/Utils and change namespace. All reusable scripts should placed there (like RotatingCube).